package.json

```diff
@@ -11,6 +11,8 @@
     "lint": "eslint .",
     "format": "prettier --write .",
     "format:check": "prettier --check ."
+    "format": "prettier --write src",
+    "format:check": "prettier --check src"
```

Comment:
unless we anticipate writing js code outside of the `src/` directory, this should be fine. I was running into issues where I'd created a venv in the `experiment` directory and installed `jupyter` into it - Jupyter gets packaged with a bunch of javascript files so prettier was running over those files as well